### DISCUSSION TITLE
Performance compared to NativeEthernet?

### DISCUSSION THREAD

**Discussion by UrielGuy** [ssilverman/QNEthernet#23] [Q&A]
Hey!
I have a project that seems to be pushing the limits of NativeEthernet, and by that I mean I see a lot of TCP Retransmissions. I was wondering (and might be interesting to put on the main page?) if you have a performance comparison, as in what is the bandwidth one can expect with the different platforms. I would have tested myself, but all of the hardware I had for this is now in storage near Burning Man. 

Appreciate this, amazing work!

Uriel

**Comment by ssilverman** [comment]
Thank you for the kind words.

I don't have a comparison and I'm unlikely to do one, but does this approximate measurement count?
https://github.com/ssilverman/QNEthernet/blob/7852eaa310dfb15fc15cc352b67b5aceff33847a/examples/IPerfServer/IPerfServer.ino#L16

**Reply by UrielGuy** [reply]
Absolutely! Thank you so much!
